demo/index.html

```diff
@@ -21,6 +21,7 @@
 	<!--<link href="../node_modules/@angular/material/prebuilt-themes/indigo-pink.css" rel="stylesheet">-->
 	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.7.0/css/perfect-scrollbar.min.css" />
```

Comment:
I guess css should be added somewhere in main app, not demo. User shouldn't think about import of these styles.